integration-tests/testutils/build/build_test.go

```diff
@@ -263,3 +264,15 @@ func (s *BuildSuite) TestAssetsDoesNotBuildSnapdFromBranchIfNotInstructedTo(c *c
 		check.Commentf("Expected 0 call to execCommand with %s, got %d",
 			buildSnapdCmd, buildCall))
 }
+
+func (s *BuildSuite) TestAssetsHonoursBuildTags(c *check.C) {
+	testBuildTags := "mybuildtag"
+	Assets(s.useSnappyFromBranch, s.arch, testBuildTags)
```

Comment:
Please note this specific branch is not blocked on that point, since this was already there before, so feel free to merge. I would still look for ways to improve nomenclature, but this isn't worth our time debating it much further. As a suggestion, Go has a `build` package too (http://golang.org/pkg/go/build), so it might be interesting to consider the difference between `build.Assets` and `build.Context` in the two packages.

But again, that's an unrelated point. Please feel free to go ahead with this branch.